controllers/limited_support/limited_support_controller_test.go

```diff
@@ -26,19 +27,22 @@ func makeTestConfigMap(name string, namespace string) *corev1.ConfigMap {
 
 func TestReconcileLimitedSupportConfigMap_Reconcile(t *testing.T) {
 	for _, tc := range []struct {
-		name            string
-		expectedResults string
+		name             string
+		expectedResults  string
+		testEnvClusterID string
 	}{
 		{
-			name: "limited-support correct ConfigMap",
+			name:             "limited-support correct ConfigMap",
+			testEnvClusterID: "i-am-a-cluster-id",
 			expectedResults: `
 # HELP limited_support_enabled Indicates if limited support is enabled
 # TYPE limited_support_enabled gauge
-limited_support_enabled{name="osd_exporter"} 1
+limited_support_enabled{_id="i-am-a-cluster-id",name="osd_exporter"} 1
 `,
 		},
 	} {
 		t.Run(tc.name, func(t *testing.T) {
+			os.Setenv(EnvClusterID, tc.testEnvClusterID)
```

Comment:
Let's use `t.Setenv` here, the difference is small, but `os.Setenv` doesn't clean up after itself, so running these tests locally could start leaking information through environment variables to other tests.

```suggestion
			t.Setenv(EnvClusterID, tc.testEnvClusterID)
```